client/src/components/Categories/CategoryCard.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { CardLink, Row, Card, CardText, CardBody, CardTitle, Button, CardImg, Col } from 'reactstrap';
 import { Link } from 'react-router-dom';
+import CategoryIcon from './CategoryIcon';
```

Comment:
why did you change this file `CategoryCard.js` - this shouldn't have changed.

Reply:
I'll fix it now and send it

Reply:
Mozafar I just added the import at the top because when I started git pull upstream it was already in the file categoryCard.js